plugins/obs-webrtc/whip-output.cpp

```diff
@@ -148,16 +147,24 @@ void WHIPOutput::ConfigureVideoTrack(std::string media_stream_id,
 		ssrc, cname, video_payload_type,
 		rtc::H264RtpPacketizer::defaultClockRate);
 
-	if (is_av1) {
+	if (strcmp("av1", codec) == 0) {
```

Comment:
Just a nitpick, I would like to see these compares in the following order: `h264`, `hevc`, `av1`